WordPress/src/debug/AndroidManifest.xml

```diff
@@ -18,6 +18,9 @@
     <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
     <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
 
-    <!-- Allows changing locales -->
-    <uses-permission android:name="android.permission.CHANGE_CONFIGURATION" />
+    <!-- Allows changing locales for screenshot automation -->
+    <uses-permission
+        tools:ignore="ProtectedPermissions"
```

Comment:
Adding `tools:ignore="ProtectedPermissions"` here tells the linter to ignore errors with this permission. Since this is only in the debug manifest I think this is safe to do.

Reply:
And we only use this permission in Fastlane to adjust the settings for writing screenshots out, so I agree – shouldn't be a problem!